modules/monitoring-assigning-tolerations-to-monitoring-components.adoc

```diff
@@ -3,54 +3,75 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="assigning-tolerations-to-monitoring-components_{context}"]
-= Assigning tolerations to monitoring components
 
-ifndef::openshift-dedicated,openshift-rosa[]
+// The final solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+// tag::CPM[]
+[id="assigning-tolerations-to-monitoring-components-cpm_{context}"]
+= Assigning tolerations to monitoring components for core platform monitoring
+// end::CPM[]
+
+// tag::UWM[]
+[id="assigning-tolerations-to-monitoring-components-uwm_{context}"]
+= Assigning tolerations to monitoring components for monitoring for user-defined projects
+// end::UWM[]
+
+// Set attributes to distinguish between cluster monitoring example (core platform monitoring - CPM) and user workload monitoring (UWM) examples.
+// tag::CPM[]
+:configmap-name: cluster-monitoring-config
+:namespace-name: openshift-monitoring
+:component: alertmanagerMain
+// end::CPM[]
+// tag::UWM[]
+:configmap-name: user-workload-monitoring-config
+:namespace-name: openshift-user-workload-monitoring
+:component: thanosRuler
+// end::UWM[]
+
+// tag::CPM[]
 You can assign tolerations to any of the monitoring stack components to enable moving them to tainted nodes.
-endif::openshift-dedicated,openshift-rosa[]
+// end::CPM[]
 
-ifdef::openshift-dedicated,openshift-rosa[]
+// tag::UWM[]
 You can assign tolerations to the components that monitor user-defined projects, to enable moving them to tainted worker nodes. Scheduling is not permitted on control plane or infrastructure nodes.
-endif::openshift-dedicated,openshift-rosa[]
+// end::UWM[]
 
 .Prerequisites
 
+// tag::CPM[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
```

Comment:
This one is appearing in the ROSA distribution. I'm not sure that ROSA/OSD users need to create this config map. For example, the user-workload-monitoring-config config map is created by default.

Reply:
Oh yes, I was adding the ifdef statements for the files with CPM tagging initially, but that got too confusing for the reviewers, so I defided not to include it for now, as the modules will move around anyway. The CPM files will be ultimately in its separate assembly that will not be included in the ROSA/OSD config map. Thank you for asking! :slightly_smiling_face: